HOWTO.md

```diff
@@ -594,3 +601,136 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
+
+You can run the `fdo-manufacturing-client` using the [provided
+CLI mode](#cli-mode) or setting up [environment
+variables](#environment-variables-mode).
+
+For both cases, the Manufacturing client will only run if no Device Credentials
+are found in any of the default locations:
+`/sys/firmware/qemu_fw_cfg/by_name/opt/device_onboarding/devicecredential/raw`,
+a location previously set up by the `DEVICE_CREDENTIAL` environment variable,
+or `/etc/device-credentials`.
+
+#### Environment variables mode
+
+Please note that the environment variables shown in this section and
+subsections are *required* unless said otherwise. 
+
+* `MANUFACTURING_SERVER_URL`: URL of the manufacturing server.
+* `USE_PLAIN_DI`: [optional] sets the Device Identification mode to `plain-di`
+  or `no-plain-di`, by default `no-plain-di`.
+  
+The Manufacturing client will then operate in one of these two different modes:
+`plain-di` or `no-plain-di`, which require a different set of environment
+variables: 
+
+##### `no-plain-di`
+
+With this mode the Manufacturing server will request a specific Device
+Identification method to the device.
+
+1. Select the DIUN Public Key Verification Mode using *one* of the following
+   environment variables and configuring it with the required value:
+   
+   - `DIUN_PUB_KEY_ROOTCERTS`: X509 certificate-based DIUN Public Key
+     Verification Mode. Requires a path to the certificate.
+   - `DIUN_PUB_KEY_HASH`: hash-based DIUN Public Key Verification
+     Mode. Available options: `sha256` or `sha384`.
+   - `DIUN_PUB_KEY_INSECURE`: (boolean) sets Public Key Verification Mode to
+     `insecure`.
+   
+   If more than one environment variable is set the first one in the following
+   order will take precedence: `DIUN_PUB_KEY_ROOTCERTS`, `DIUN_PUB_KEY_HASH`
+   and `DIUN_PUB_KEY_INSECURE`.
+   
+2. If the Manufacturing server is specifically configured with a
+   `mfg_string_type` set to `MACAddress` in its `diun` configuration section it
+   will ask the device to identify itself with a MAC Address, in order to do so
+   the device must set `DI_MFG_STRING_TYPE_MAC_IFACE` [optional] to a valid
+   interface.
+   
+   Please not that this `DI_MFG_STRING_TYPE_MAC_IFACE` environment variable is
+   *optional* and will only be read if the server requests a `MACAddress`
+   identification mode, but if the Manufacturing server is configured as
+   described above the Device Identification protocol will result in an early
+   error if this environment variable is not set.
+   
+   In such case, the value `DI_MFG_STRING_TYPE_MAC_IFACE` must be set to any
+   interface name of the device that does not result in `00:00:00:00:00:00` as
+   the specified MAC Address.
+   
+3. Run the client: `fdo-manufacturing-client`.
+
+##### `plain-di`
+
+The Device will have to configure all the required options for its
+identification with the Manufacturing server.
+
+1. `DI_MFG_STRING_TYPE`: [optional] selects the Device Identification string
```

Comment:
This is `Device Identification` and it should be ok as it identifies the device.